### DISCUSSION TITLE
Certificate Request Error: Certificate provisioning will retry automatically in a short period, please be patient. (on sub domin)

### DISCUSSION THREAD

**Discussion by phantomboy0** [community/community#49244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i added my subdomin ( amir.us.to ) to github pages and it says  (Certificate Request Error: Certificate provisioning will retry automatically in a short period, please be patient. ) for 2 and half days now
i did change CNAME of my subdomin to my github page (phantomboy0.github.io ) and i can get redirected when i type my subdomin in my browser but i need tha sweet HTTPS .
help.

**Comment by yoannchaudet** [comment]
Have you requested a lot of TLS certificates from this domain in the past few days?

It looks like you are being rated limited by Let's Encrypt (which we use for certificates).

https://letsdebug.net/amir.us.to/1397161

![image](https://user-images.githubusercontent.com/14911070/223027546-72eddadd-77e3-4634-be37-9f05b3a63315.png)

**Reply by phantomboy0** [reply]
yse the main domain us.to was rate limited so as my amir.us.to subdomin
i changed the subdomain provider and it was fixed
